.github/workflows/shaders.yml

```diff
@@ -3,8 +3,8 @@ name: Shaders
 on:
   push:
     branches-ignore: [
-        # We don't need to run on dependabot PRs.
-        "dependabot/**",
+        # We don't need to run on renovate PRs.
+        "renovate/**",
```

Comment:
question(non-blocking): It doesn't really _hurt_ to keep `dependabot` configuration here, doesn't it? 🤔

Reply:
I thought about, and just didn't. I kinda figured we wouldn't go back, and I think that assessment was correct.